.github/workflows/build-publish.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Build Publish Chainlink'
+
+on:
+  push:
+    branches:
+      - 'release/**'
+
+jobs:
+
+  build-publish-chainlink:
+    # buildjet-2vcpu-ubuntu-1804
+    runs-on: ubuntu-1804
```

Comment:
This is a very old version of Ubuntu, might we want to use 20.04.3 LTS instead?

Reply:
I think `ubuntu-latest` uses Ubuntu 20.04 LTS so it should be good

Reply:
https://github.blog/changelog/2020-10-29-github-actions-ubuntu-latest-workflows-will-use-ubuntu-20-04/